manifests/n/N1MM/Logger+/1.0.10001/N1MM.Logger+.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: N1MM.Logger+
+PackageVersion: 1.0.10001
+InstallerType: nullsoft
+ProductCode: "N1MM Logger+"
+Installers:
+- InstallerUrl: https://n1mmwp.hamdocs.com/mmfile/get/file/N1MM-Logger-FullInstaller-1.0.10001.exe
```

Comment:
This URL blocks users from visiting it directly and response HTTP 403 Forbidden. We might not able to submit this package to the repository. 😞